README.md

```diff
@@ -25,7 +25,7 @@
 ## Installation
 
 ```bash
-$ npm i --save @nestjs/swagger
+$ npm i --save @nestjs/swagger swagger-ui-express

```

Comment:
This depends on an HTTP driver your app uses

Reply:
I thought I'd add only that as `@nestjs/platform-express` (Express) is used by default. However I remove it I leave it as before